statistics/most_distinct_dates_competed_on.rb

```diff
@@ -3,20 +3,22 @@
 class MostDistinctDatesCompetedOn < Statistic
   def initialize
     @title = "Most distinct dates competed on"
-    @table_header = { "Dates" => :right, "Person" => :left}
+    @table_header = { "Dates" => :right, "Person" => :left, "List" => :left }
   end
 
   def query
     <<-SQL
       SELECT
         attended_dates,
-        CONCAT('[', person.name, '](https://www.worldcubeassociation.org/persons/', person.id, ')') person_link
+        CONCAT('[', person.name, '](https://www.worldcubeassociation.org/persons/', person.id, ')') person_link,
+        dates_list
       FROM (
         SELECT
-          COUNT(DISTINCT DATE_FORMAT(competition_date, '%m%d')) AS attended_dates,
-          personId
+          COUNT(DISTINCT competition_date) AS attended_dates,
+          personId,
+          GROUP_CONCAT(DISTINCT competition_date ORDER BY competition_date ASC SEPARATOR ', ') dates_list
         FROM (
-          SELECT Results.personId, competition_dates.competition_date
+          SELECT Results.personId, DATE_FORMAT(competition_dates.competition_date, '%m%d') competition_date
```

Comment:
```suggestion
          SELECT Results.personId, DATE_FORMAT(competition_dates.competition_date, '%m/%d') competition_date
```

Reply:
Although ideally we would post-process this into something more readable, e.g. `January: 1, 2, 5, 12, 30; February: 4, 14, 18; April: ...` using `def transform` :)

Reply:
Yeah I thought about doing that but it started getting late and I have to work tomorrow. How about if I make the first change now and start looking at part 2 for a couple of days from now?

Reply:
Sure, whenever you feel like it :)